moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -142,12 +142,23 @@ class Robot
                         std::vector<double> & joint_point) const;
 
   /**
-   * @brief execute a given trajectory
+   * @brief plans a given trajectory
    * @param traj_name - name of trajectory to be executed (must be filled with
    * prior calls to "addTrajPoint".
-   * @return - true if desired trajectory was executed.
+   * @param goal - control_msgs::FollowJointTrajectoryGoal
+   * @return - true if desired trajectory was planned
    */
-  bool execute(const std::string traj_name);
```

Comment:
Actually, after seeing your tests, I like usability of this method.  The "new" methods give you granular control for testing and other uses, but we still need this method for simplicity.